docs/self_sign_cert.md

```diff
@@ -0,0 +1,51 @@
+# Accessing private docker registry with self signed certificate
+
+Currently, `jib` do not support docker registries with self signed `https` certificate.
+
+The only way to get `jib` working is to import the self signed certificate into jvm `CA Certificates Keystore`.
```

Comment:
How about:
> Jib uses the JRE's list of approved _CA Certificates_ to validate SSL certificates.  The following instructions describe how to add a registry's self-signed certificate to the JRE's approved CAs.

Reply:
How about also adding

"The certificate will be trusted at the JRE level, affecting all Java applications running on it. You will also need to re-import the certificate when you use a different JRE or upgrade it."